static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx

```diff
@@ -193,7 +193,7 @@ function SimilarStackTrace({project}: Props) {
       {status === 'ready' && !hasSimilarItems && !hasSimilarityEmbeddingsFeature && (
         <Panel>
           <EmptyStateWarning>
-            <p>{t("There don't seem to be any similar issues.")}</p>
+            <Title>{t("There don't seem to be any similar issues.")}</Title>
```

Comment:
This:
<img width="458" alt="image" src="https://github.com/user-attachments/assets/37ef0cad-6509-4064-b281-9aaf1b3977af">

vs this:
<img width="446" alt="image" src="https://github.com/user-attachments/assets/6d331294-0c7c-4eaa-b537-6b35715b8f7d">

Reply:
We should change the font for similarity embeddings as well [here](https://github.com/getsentry/sentry/pull/81177/files#diff-ed4ce224039f8283b042b51a4e06a71c416182eda6b4a5185e4ba3faa72ce904R205) to match

Reply:
We should change the font for similarity embeddings as well [here](https://github.com/getsentry/sentry/pull/81177/files#diff-ed4ce224039f8283b042b51a4e06a71c416182eda6b4a5185e4ba3faa72ce904R205) to match